src/__tests__/application/useCase/DeletePromptUseCase.test.ts

```
@@ -0,0 +1,42 @@
+import { deletePromptUseCase } from "../../../application/useCase/DeletePromptUseCase";
+
+import { deletePromptUseCase } from "../../../application/useCase/DeletePromptUseCase";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'deletePromptUseCase'. Consider to delete it or rename it.

'deletePromptUseCase' is defined here:


(lint/suspicious/noRedeclare)